source/common/config/utility.h

```diff
@@ -222,6 +223,26 @@ class Utility {
     return {ALL_SUBSCRIPTION_STATS(POOL_COUNTER(scope), POOL_GAUGE(scope))};
   }
 
+  /**
+   * Generate an IncrementalSubscriptionStats object from stats scope.
+   * @param scope for stats.
+   * @return SubscriptionStats for scope.
+   */
+  static IncrementalSubscriptionStats generateIncrementalStats(Stats::Scope& scope) {
```

Comment:
Why can't we share stats for incremental/non-incremental?

Reply:
Done.